### PR TITLE
Fix typo in chapter 2

### PR DESCRIPTION
**Pull request by proudmuslim-dev**

### CODE REVIEW

File: blog/content/edition-3/posts/02-booting/index.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ UEFI, in contrast, is more modern and has much more features, but also more comp
 ### BIOS
 
 Almost all x86 systems have support for BIOS booting, including most UEFI-based machines that support an emulated BIOS.
-This is great, because you can use the same boot logic across all machines from the last centuries.
+This is great, because you can use the same boot logic across all machines from the last century.
```
Comment (phil-opp) [inline blog/content/edition-3/posts/02-booting/index.md:59]: Thanks! BIOS is around for longer than one century, so maybe "from the last few centuries" instead?

Comment (infernostars) [reply blog/content/edition-3/posts/02-booting/index.md:59]: Theoretically, "last couple of millenia" would be technically correct [in the best way possible]

Comment (proudmuslim-dev) [reply blog/content/edition-3/posts/02-booting/index.md:59]: I was taking "century" literally (i.e. as in the last 100 years)- not really sure what the best way to word this unambiguously would be

Comment (phil-opp) [reply blog/content/edition-3/posts/02-booting/index.md:59]: Ah sorry, I confused century with decade :D.

**Pull request merged**